mobile_verifier/src/speedtests_average.rs

```diff
@@ -0,0 +1,478 @@
+use crate::speedtests::{self, Speedtest};
+use chrono::{DateTime, Duration, Utc};
+use file_store::{file_sink::FileSinkClient, traits::TimestampEncode};
+use helium_crypto::PublicKeyBinary;
+use helium_proto::services::poc_mobile::{self as proto, SpeedtestAvgValidity};
```

Comment:
```suggestion
use helium_proto::services::poc_mobile::proto;
```
Looks like you’re (typically) referencing the SpeedtestAvgValidity as `proto::SpeedtestAvgValidity` anyway so you can clean up this import a bit

Reply:
addressed here: 67effc3558d2cd2c56ef07956dba0b19b02ffd95